src/main/scala/gitbucket/core/controller/SystemSettingsController.scala

```diff
@@ -116,8 +127,8 @@ trait SystemSettingsControllerBase extends AccountManagementControllerBase {
       if (settings.ssh.enabled && settings.baseUrl.isEmpty) {
         Some("baseUrl" -> "Base URL is required if SSH access is enabled.")
       } else None,
-      if (settings.ssh.enabled && settings.ssh.sshHost.isEmpty) {
-        Some("sshHost" -> "SSH host is required if SSH access is enabled.")
+      if (settings.ssh.enabled && settings.ssh.bindAddress.isEmpty) {
```

Comment:
Looks like `publicAddress` is also required if we want to enable SSH.

Reply:
Sorry, I overlooked that `sshBindAddress` is used as `sshPublicAddress` if `sshPublicAddress` is not given.